daml-lf/encoder/src/test/lf/Decimal_all_.lf

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
why is automation missing these files?

Reply:
`lf` is not in the list of supported extensions.

Reply:
Can we add it? Or is that some thirdparty dep that we cannot easily change?

Reply:
I'll see what I can do. I've been reluctant to touch a script that only really annoys me once a year, but this should be simple enough. Separate PR, though.

Reply:
#12259